core/iwasm/aot/aot_loader.c

```diff
@@ -1169,17 +1176,30 @@ load_init_expr(const uint8 **p_buf, const uint8 *buf_end, AOTModule *module,
             read_uint64(buf, buf_end, expr->u.i64);
             break;
         case INIT_EXPR_TYPE_V128_CONST:
-            read_byte_array(buf, buf_end, &expr->u.v128, sizeof(expr->u.v128));
+            i64x2 = (uint64 *)expr->u.v128.i64x2;
+            CHECK_BUF(buf, buf_end, sizeof(uint64) * 2);
+            wasm_runtime_read_v128(buf, &i64x2[0], &i64x2[1]);
+            buf += sizeof(uint64) * 2;
             break;
         case INIT_EXPR_TYPE_GET_GLOBAL:
             read_uint32(buf, buf_end, expr->u.global_index);
             break;
+#if WASM_ENABLE_GC == 0
+        case INIT_EXPR_TYPE_FUNCREF_CONST:
         case INIT_EXPR_TYPE_REFNULL_CONST:
-            read_uint32(buf, buf_end, expr->u.type_index);
+#if UINTPTR_MAX == UINT64_MAX
+            read_uint64(buf, buf_end, expr->u.ref_index);
+#else
+            read_uint32(buf, buf_end, expr->u.ref_index);
```

Comment:
Found that `void *data;` field of WASMValue is added in `core/iwasm/gc_export.h`, it isn't added in WASMValue definition in `core/iwasm/interpreter/wasm.h`, could you help add it?